language/move-prover/tests/sources/functional/script_incorrect.exp

```diff
@@ -8,10 +8,10 @@ error: abort not covered by any of the `aborts_if` clauses
 14 │ │ }
    │ ╰─^
    │
-   ┌─ tests/sources/functional/script_provider.move:15:9
+   ┌─ tests/sources/functional/script_provider.move:16:9
    │
-15 │         assert!(signer::address_of(account) == @0x1, 1);
```

Comment:
Why this error gets changed?

Reply:
Basically, the test case seems to be bad because it allows nondeterministic counter-examples. I updated it making it deterministic.